components/com_users/tmpl/registration/default.php

```diff
@@ -34,10 +35,19 @@
 					<?php if (isset($fieldset->label)) : ?>
 						<legend><?php echo Text::_($fieldset->label); ?></legend>
 					<?php endif; ?>
-					<?php echo $this->form->renderFieldset($fieldset->name); ?>
+					<?php foreach ($fields as $field): ?>			
```

Comment:
```suggestion
					<?php foreach ($fields as $field): ?>
```

remove tabs at end of line